### PR TITLE
[DOCS] Add link to panelite in the secondary sidebar

### PR DESCRIPTION
**Pull request by maximlt**
Going for ease of implementation with this one, adding a link to Panelite in the secondary sidebar.

*Example from the playground website of nbsite*
image

I haven't checked the generated links, I think they're going to be right but I'm going to be sure about it only when I see the site deployed :)

I went for `Open in Panelite` as this is technically what it does, however I'm pretty sure that's not a great message, most people won't know what Panelite is or means and I'm not sure we want them to learn that? A common UX for launching stuff in a computing environment is to use a badge.

image

Should we go for a badge with something like `launch: jupyterlite`? Suggesting jupyterlite as that at least includes jupyter which I expect people to know about.

### CODE REVIEW

**Comment by Hoxbro** [comment]
Jupyterlite has a launch icon for what it is worth: https://github.com/jupyterlite/jupyterlite/blob/main/docs/_static/badge-launch.svg

**Comment by maximlt** [comment]
> Jupyterlite has a launch icon for what it is worth: https://github.com/jupyterlite/jupyterlite/blob/main/docs/_static/badge-launch.svg

Adding a screenshot to the badge shared by Simon:

image

I quite like that badge, even if I would prefer replacing `lite` with `jupyterlite` to be more explicit.

**Comment by philippjfr** [comment]
Badge sounds great, and I agree that if we could get the `jupyterlite` wording in that would be even better.

File: doc/_templates/panelitelink.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{# Custom template to include a link to Panelite #}
+{% if title and pagename != root_doc %}
+  <div class="tocsection">
+    <a href="{{ panelite_endpoint + '/lab?path=' + pagename + '.ipynb' }}" target=”_blank”>
```
Comment (philippjfr) [inline doc/_templates/panelitelink.html:4]: `pagename` includes the path, e.g. `how_to/styling/design`?

**Comment by Hoxbro** [comment]
> I quite like that badge, even if I would prefer replacing lite with jupyterlite to be more explicit.

Me too. I tried to change it with Inkscape but gave up because of some mask problem.

File: doc/_templates/panelitelink.html (hunk 1 above)
Comment (maximlt) [reply doc/_templates/panelitelink.html:4]: That's my observation.

**Comment by maximlt** [comment]
This is how the badge looks like. I hope that yellow won't be too much of a mismatch with the new blue accent adopted across the site.

image

How I've created it:
- Took the jupyerlite icon from https://github.com/jupyterlite/jupyterlite/blob/2ee9cba529cb514c00130924c355d8696fd4562c/docs/_static/icon.svg
- Turned it into a PNG (https://svgtopng.com/)
- Turned the PNG into a base64 string (https://www.base64-image.de/), I tried originally with the SVG but the generated base64 string was too long for shields.io
- Adapted the link generated on this website to our use case, by changing the text and setting the color to `#f7dc1e` which is the Jupyterlite yellow (https://mybinder.readthedocs.io/en/latest/howto/badges.html)

The URL used to hit `img.shields.io` and generate the SVG badge:
```
https://img.shields.io/badge/launch-jupyterlite-f7dc1e.svg?logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEQAAABECAYAAAA4E5OyAAAABmJLR0QA/wD/AP+gvaeTAAAHCUlEQVR42u2beWwUVRzHa9QgXiT+g+7Mm6GltRwtSEERNQi2EEC8AoKWK7uFJmoJoLLrhTRKxHAGFY3YQzmloiY0QCHlKoI04bIctsxOCwgIAZGj3V1UeL7fvO5N2aX7m93ZTSf5ptOZ2Z33Pn2/3/v9fu81KantMMZB9/S+01krJzfZpd5OpdMAkHbOrsG9hAfgrE+WnYo80alIy512uc5pl/5hoi0I7tVqzypynlNNkRJjFDR0usupyhbWuSqm6zcBEErXnaq03akSM1VS28UfiD/E9g67/DbryJ8RQGhJp9l3vwWw48M0VClbG+74IAJld6nyMEObh8MuLYoCiEAtpadNdxtsVKRIrGE1MYChyaVK+x12EzEEjKtH5a6sUSdiBcNHp67Wiz1iCsNlF1NZQ84YAIZb51z1yekxgdFUS0ysAQ0GgsGlSvWNDZ0ejK4D3Zp0R3NsQY0peRulSbdHz4na5VnGhdEsRSqMjhO1k4wQYbdR9O9VVczU11Ro0m3sRTuwGn3hkN5QyFadZxV5KEZDD20mNOdpgZpMJtq3t0CrfiR6xig5OvoOHEc6NNukwXCrR4ZAr9Tp52D19B0RN7CRdZwQfyCgw1v0GyW6+BKXnczDauCgAYIfjK5dTPTS78HPnashdNsaQjevJvT8wQjMRpE+1cNc0MLzfRWEPpbFYWR0E+imVcGjY80SQruke6FldDPR6vLWB2u4o0MV0/QYyseruQkFXq/8ntzQrEYMF1r/PlaaxBsdUPpDhrGgUKQLmQKvX66VaK8eHMD4lwVaVyXRb+YR7ffuXU0RhPSyBQ0Idp3j4GbeQUEI9h1LF4navUezBPpXc5zy61r+PIBq/fQrL8D0HxWYQH4q4h1MTws2geeHcIdbPN/rV0oWcEhwL4L3rsMEcggTyPyZHMjggf4dPHtAohLzHaJooieqvUDAd8Dzc2eIkby3BhEIOY4JpMDCOzhxjD+Qn4s5qCf7ek0DZhYwLYB0ZGtE723AHCHnMYE8O4gDmWXz/4t/ZOWmMWEUB9V0VKIvDOXPWnLFiItHmEBOYQLp1oXPIuBAfa/DiIHrM6bx61/P4YBkyaQ54gjfexKz/lGHBePsfskTV2wPSOqGZHMg82eKdM96Qjun8OdmvydivPsI5ghBS/ndUygIAjPfe1k9+fWPmek8ksnPwbwgNkEoLW5HDMxIMRaQlYu5GcBf36H43+vTS/CYCPzsmRkMLQItQQzM5OlYQD55lwMZ+JT/DLN/o0RTO3uTPshjqsvxMmBY/sQE8gRWwyaN5Z2eMNoLZNlnhKYke3OWzO6ClgBiOnLHUbEvdpX9Mmbq/+GbIr14RKJTJ4keEGmp/Oc7BSJ2InkZ+oCc8Uo/YDTu4VTB4ziH+FTOxo0UGCR+D64jV81W4ReIFOk5jHQ/MKWHNB+yXnCw5d8Rj1P9+zBiXZXVgvXZAsWCm0gaBtUvXxgQoPkWh87s47kM3FtbiuZDTqCbi8e5KmRaaxsGdQ1I5twwIAhTdwZ3+pUX+TN5uQLW6CjQb13mt473tGZXECRt6WnekfF4H+GGNVTQhuV8FEEy19K0C8443HAddjPpu4ypSGPDBQGdhhkj0G+AA73Z59wJHUSrvlB2l/PwPpUFdcd2hzHVKtIo/Re6+epdZTg5y0vDvCaS3V/Qqutw/v6UmwNRdhBP6A6pP3x2WI6gjRp3KTGMVb+KqC12N9YndwxlOm5fAHrDLGrLCO6QfMnc0HFGTSUJWtAC9e8n0L0bQjrcs7BdI7qbZdgSYUuL3lDtcncgf5ygTanuOipo48rwZhD43JYyok3Lcz4QtWm5MfQKH2uTPDBGe8vkXNaAa4GNOrDJ2/nln/POfznb60vqqnRbpYO9sBNiu61KIa8HQoF0Pa2zN2Odli9qBWX3DBOY5SLpmkslrxli453DTkazBrl8G1i6MHh2Aa34QpfR4WKQRxpqa2aTImfBZlrfhq79ltDhgwU2SgTNQa7+StQDxjHUTBZ1SlZTOrD96EUR7msP20Sg6MP2vt9v+O3dDkXuxxq7S0cYvxh2VIQoTj/D1nTWsw78hwABvmNdzKZUVP9yTH7IoUpTm+FcuQUI8Ow6FvNMifre06j5GVZGgB09MCvAv5DAZhaHKi8GaefsGjsfAc/olrbH6igpKSlgushEEXSNaWdRUdED8QzkDBIMj0pLSy3xDOSkDkDGxS2Q4uLiEawTB5hUBNmZVpSVlbVPajsS5LBarWar1Xaa6QKSKtl33hfHQGzHmSiurGPagPjIZrPlxi2Q6dNt41kHTiKaTEVhYeG9bc4oUY/Jkye3M+flLWOymy0WNYT2WiyWnIQGYjabR7KO0lvQ3kQH8uotAqlJaCD5+fkdWCd3MV0PA8YlBnBsm6NpO4x1/A+osdHK73zSVAAAAABJRU5ErkJggg==
```

File: doc/_templates/panelitelink.html (hunk 1 above)
Comment (philippjfr) [reply doc/_templates/panelitelink.html:4]: 👍

**Comment by philippjfr** [comment]
>I hope that yellow won't be too much of a mismatch with the new blue accent adopted across the site.

Blue and yellow are pretty complementary.

**Review by philippjfr — APPROVED** [review]
Looks good, let's see how it goes.

**Pull request merged**